warehouse/views.py

```diff
@@ -325,19 +326,34 @@ def filter_key(item):
         origin_cache(1 * 24 * 60 * 60),  # 1 day
     ],
 )
+@view_config(
+    route_name="stats.json",
+    renderer="json",
+    decorator=[
+        cache_control(1 * 24 * 60 * 60),  # 1 day
+        origin_cache(1 * 24 * 60 * 60),  # 1 day
+    ],
+    accept="application/json",
+)
 def stats(request):
     total_size_query = request.db.query(func.sum(File.size)).all()
+    total_size_human = format_size(parse_size(str(total_size_query[0][0])))
     top_100_packages = (
         request.db.query(File.name, func.sum(File.size))
         .group_by(File.name)
         .order_by(func.sum(File.size).desc())
         .limit(100)
         .all()
     )
+    top_100_packages_human = {}
+    for pkg_name, pkg_bytes in top_100_packages:
+        top_100_packages_human[pkg_name] = format_size(parse_size(str(pkg_bytes)))
 
     return {
         "total_packages_size": total_size_query[0][0],
+        "total_packages_size_human": total_size_human,
```

Comment:
Instead of adding a parameter here, we should just return the bytes in the dictionary. In the template we can convert to using a human friendly display by doing ``total_packages_size|filesizeformat()``. This is what we do eslewhere:

https://github.com/pypa/warehouse/blob/6e8aebc1bc9fc01def59c877387b21a54b0c62b7/warehouse/templates/packaging/detail.html#L251